js_basics/task5.js

```diff
@@ -0,0 +1,14 @@
+import chalk from 'chalk';
+
+const radius = 4;
+const length = 35;
+const width = 12;
+const height = 10;
+
+const circleArea = (Math.PI * (radius ^ 2)).toFixed(2);
```

Comment:
`(radius ^ 2) `- this operator is bit-wise XOR - it does different thing
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_XOR
)
You can compare results
```js
console.log(radius ^ 2)
console.log(radius ** 2)
console.log(Math.pow(radius,2))
```

Reply:
fixed